content/en/docs/installation/operator.md

```diff
@@ -116,125 +64,73 @@ NAMESPACE                             READY         STATUS       RESTARTS      A
-   subjects:
-   - kind: ServiceAccount
-     name: spinnaker-operator
-     namespace: spinnaker-operator # Edit this value if you want Operator to live in a different namespace.
-   roleRef:
-     kind: ClusterRole
-     name: spinnaker-operator-role
-     apiGroup: rbac.authorization.k8s.io
-   ```
 
-    Then, save the changes to `role_binding.yaml`.
+## Spinnaker Install
 
-3. Run the following command:
-
-   ```bash
-   $ kubectl apply -n <op_namespace> -f deploy/operator/cluster
-   ```
-
-   `<op_namespace>` is the namespace where you want the operator to live. By default, this namespace is `spinnaker-operator`, so you would run the following command:
-
-   ```bash
-   $ kubectl apply -n spinnaker-operator -f deploy/operator/cluster
-   ```
-
-After installation, you can verify that the Operator is running with the following command:
+To use the examples, change the parameters you need (especially the `persistentStorage` section). To install a basic version of Spinnaker in the "spinnaker" namespace, run the following command:
 
 ```bash
-kubectl -n <op_namespace> get pods
-```
-The command returns output similar to the following if the pod for the Operator is running:
+kubectl create ns spinnaker
+kubectl -n spinnaker apply -f deploy/spinnaker/basic
 
-```
-NAMESPACE                                READY         STATUS       RESTARTS      AGE
-spinnaker-operator-7cd659654b-4vktl      2/2           Running      0             6s
+# Watch the install progress and check out the pods being created too!
+kubectl -n spinnaker get spinsvc spinnaker -w
 ```
 
-## Installing Spinnaker Using Operator
+### How it works
+Spinnaker's configuration can be found in a `spinnakerservices.spinnaker.armory.io` Custom Resource Definition (CRD) that can be stored in version control. After you install Spinnaker Operator, you can use `kubectl` to manage the lifecycle of your deployment.
 
-Once you install the CRDs and Operator, check out the examples in `deploy/spinnaker/`. To use the examples, change the parameters you need (especially the `persistentStorage` section). To install a basic version of Spinnaker with Operator, run the following command:
-
-```bash
-$ kubectl create ns <spinnaker-namespace>
-$ kubectl -n <spinnaker-namespace> apply -f deploy/spinnaker/basic/SpinnakerService.yml
+```yaml
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    config:
+      version: 2.21.0
 ```
-In the examples, the `spinnaker-namespace` parameter refers to the namespace where you want to install Spinnaker. It is likely different from Operator's namespace.
 
-**Important**: You must edit `deploy/spinnaker/basic/SpinnakerService.yml` to point to persistent storage, such as an S3 bucket. Other attributes can also be changed. For example, if you change the value of the `version` field to `2.16.0`, Operator installs version 2.16.0.
+See [the full format](../operator-reference)
```

Comment:
Feels like we should link to the patch files repo so that users can see some common setups, in addition to the raw operator reference:
https://github.com/armory/spinnaker-kustomize-patches